cmd/scw/testdata/test-main-usage-usage.golden

```diff
@@ -33,7 +33,6 @@ AVAILABLE COMMANDS:
   version       Display cli version
   vpc           VPC API
   vpc-gw        VPC Public Gateway API
-  help          Help about any command
```

Comment:
Why there is useful content about it:

```
$ scw help
Get help about how the CLI works

USAGE:
  scw help <command>

AVAILABLE COMMANDS:
  date        Get help about how date parsing works in the CLI
  output      Get help about how the CLI output works

FLAGS:
  -h, --help   help for help

GLOBAL FLAGS:
  -c, --config string    The path to the config file
  -D, --debug            Enable debug mode
  -o, --output string    Output format: json or human, see 'scw help output' for more info (default "human")
  -p, --profile string   The config profile to use

Use "scw help [command] --help" for more information about a command.

```

Reply:
```
AVAILABLE COMMANDS:
  account       User related data
  apple-silicon Apple silicon API
  autocomplete  Autocomplete related commands
  baremetal     Elastic metal API
  config        Config file management
  container     Container as a Service API
  dns           DNS API
  feedback      Send feedback to the Scaleway CLI Team!
  fip           Flexible IP API
  function      Function as a Service API
  help          Get help about how the CLI works
  iam           IAM API
  info          Get info about current settings
  init          Initialize the config
  instance      Instance API
  iot           This API allows you to manage IoT hubs and devices
  k8s           Kapsule API
  lb            This API allows you to manage your load balancer service
  marketplace   Marketplace API
  mnq           This API allows you to manage Messaging or Queueing brokers
  object        Object-storage utils
  rdb           Database RDB API
  redis         Managed Database for Redis™ API
  registry      Container registry API
  shell         Start shell mode
  tem           Tem
  version       Display cli version
  vpc           VPC API
  vpc-gw        VPC Public Gateway API
  help          Help about any command
```

Check the current usage when running `scw -h`. We can spot two `help` command

Reply:
I only removed the second one that is generated by cobra and not used when running `scw help`